docs/03.API-EXAMPLE.md

```diff
@@ -551,6 +551,47 @@ Value of x is 12
 Value of x is 17
 ```
 
+## Changing the seed of pseudorandom generated numbers
+By default srand is not called in JerryScript, causing it to always generate the same random numbers every time it is being run.
+To solve this, you have to use srand in your program with a seed.
+
+```c
+#include <string.h>
+#include "jerryscript.h"
+
+int
+main (void)
+{
+  const jerry_char_t script[] = "var a = Math.random(); print(a);";
+  size_t script_size = strlen ((const char *) script);
+
+  bool ret_value = jerry_run_simple (script, script_size, JERRY_INIT_EMPTY);
+
+  return (ret_value ? 0 : 1);
+}
+```
```

Comment:
I'm not convinced that a negative example is needed. A positive example where Math.random is used and srand is properly called should be enough with proper explanation.